modules/openapi-generator/src/main/resources/python/python-experimental/api_doc_example.mustache

```diff
@@ -7,51 +7,55 @@ from pprint import pprint
 {{#hasAuthMethods}}
 # Defining host is optional and default to {{{basePath}}}
 configuration.host = "{{{basePath}}}"
-# Create an instance of the API class
-api_instance = {{{packageName}}}.{{{classname}}}({{{packageName}}}.ApiClient(configuration))
+# Enter a context with an instance of the API client
+with {{{packageName}}}.ApiClient(configuration) as api_client:
+    # Create an instance of the API class
+    api_instance = {{{packageName}}}.{{{classname}}}(api_client)
```

Comment:
Aren't these blocks `{{#hasAuthMethods}}`/`{{^hasAuthMethods}}` identical except for setting 
 configuration.host?
Please move the identical block contents  underneath the `{{/hasAuthMethods}}` on line 20